melodies_monet/_cli.py

```diff
@@ -1160,6 +1165,246 @@ def get_aqs(
             ds.to_netcdf(dst / out_name)
 
 
+@app.command()
+def get_openaq(
+    start_date: str = typer.Option(..., "-s", "--start-date", help=f"Start date. {_DATE_FMT_NOTE}"),
+    end_date: str = typer.Option(..., "-e", "--end-date", help=f"End date. {_DATE_FMT_NOTE} {_DATE_END_NOTE}"),
+    out_name: str = typer.Option(None, "-o",
+        help=(
+            "Output file name (or full/relative path). "
+            "By default the name is generated like 'OpenAQ_<start-date>_<end-date>.nc'."
+        )
+    ),
+    dst: Path = typer.Option(".", "-d", "--dst", help=(
+            "Destination directory (to control output location "
+            "if using default output file name)."
+        )
+    ),
+    param: List[str] = typer.Option(["o3", "pm25", "pm10"], "-p", "--params", help=(
```

Comment:
I think makes more sense singular, unlike AQS and its parameter groups.
```suggestion
    param: List[str] = typer.Option(["o3", "pm25", "pm10"], "-p", "--param", help=(
```